modules/asset_import/revision.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ASSIMP_REVISION_H_INC
```

Comment:
This can go in `thirdparty` too, where the assimp files would typically expect it.